src/index.ts

```diff
@@ -1,41 +1,70 @@
-import * as core from '@actions/core'
-import {SecurityHubJiraSync} from '@enterprise-cmcs/macpro-security-hub-sync';
+import * as core from '@actions/core';
+import { SecurityHubJiraSync } from './macfc-security-hub-sync';
+
+// Utility function to get input with fallback to environment variable
+function getInputOrEnv(inputName: string, envName: string) {
+  const inputValue = core.getInput(inputName);
+  if (inputValue !== '') {
```

Comment:
Did you confirm that if `core.getInput(inputName)` cannot find an input for `inputName` it will return an empty string?

Reply:
👍